### DISCUSSION TITLE
Programnya looping saat menjalankan coding bagian PCA

### DISCUSSION THREAD

**Discussion by reniunisa** [teamalgoritma/community#468] [Q&A] [answered]
Halo,

Saya mencoba melakukan PCA dan clustering menggunakan data yang disarankan di LBB unsupervised learning, yaitu data spotify track .
File data saya download dari: https://www.kaggle.com/zaheenhamidani/ultimate-spotify-tracks-db 

Namun, programnya looping dan tidak berhenti2 eksekusi di bagian PCA (saya belum sampai ke clustering).
Mohon bantuan kakak-kakak algoritma :-)

Rmd nya saya lampirkan di sini
Terima kasih
Salam,

[Reni[

**Comment by takdirzd** [comment, marked as answer]
Halo Bu Reni...

untuk errornya, bu Reni bisa coba hilangkan untuk kolom "artist_name" dan "track_name" pada saat Data Cleansing, karena disitu tipe datanya factor, sedangkan untuk nama artis dan nama track itu sifatnya unik, jadi kurang tepat untuk dijadikan factor (kategorik).

note : yg untuk numeriknya, yang tetep tipe 'dbl' ya bu, bukan 'int'

jadi, codingannya tetap sama bu, hanya kedua kolom tersebut yang dibuang

![Screenshot 2022-11-25 203939](https://user-images.githubusercontent.com/88231577/203998010-45e50d86-a895-4ea2-8ae6-d02c097fd17e.png)
